util/src/main/java/io/kubernetes/client/Copy.java

```diff
@@ -161,4 +166,34 @@ public static void copyFileFromPod(String namespace, String pod, String srcPath,
       os.flush();
     }
   }
+
+  public void copyFileToPod(
+      String namespace, String pod, String container, File srcFile, String destPath)
+      throws ApiException, IOException {
+    // Run decoding and extracting processes
+    final Process proc =
+        this.exec(
+            namespace,
+            pod,
+            new String[] {"sh", "-c", "base64 -d | tar -xf - -C " + destPath},
```

Comment:
How will this work against a windows pod?

Reply:
Sorry for the late reply. Currently this function has such a limitation. But basically the same is true for the "kubectl cp" issue, and it doesn't seem to be an easy one.

https://github.com/kubernetes/kubernetes/issues/58512

Reply:
My file is 120Mb I tried this code,and it returned at 17Mb. How can I resolve this problem?

Reply:
> My file is 120Mb I tried this code,and it returned at 17Mb. How can I resolve this problem?

There may be a hole in your 120Mb file, you can test the file with `du -h your_file` to see the size without a hole.